tests/lenses/utils.go

```diff
@@ -0,0 +1,18 @@
+package lenses
+
+import (
+	"path"
+	"runtime"
+)
+
+// todo - document
+// todo - add path to lense file
+var SetDefaultModulePath string = getPathRelativeToProjectRoot(
```

Comment:
nitpick: I think `DefaultModulePath` fits better here

Reply:
This is a bad name, the 'name' of the module is `SetDefault`, as it sets a default value. Naming a Lense module `Default` would be even more unclear. I will try and come up with something else before review time though.

- [ ] Rename test lense module?

Reply:
Leaving as-is